assets/node.yaml

```diff
@@ -9,6 +9,10 @@ spec:
   selector:
     matchLabels:
       app: ovirt-csi-driver-node
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
```

Comment:
`RollingUpdate` with `maxUnavailable: 1` is the default.  [The bug][1] is asking for `maxUnavailable: 10%`, so rollout time remains bounded as the node count grows.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1933184#c0